crest/Assets/Crest/Crest/Scripts/Underwater/UnderwaterRenderer.Effect.cs

```diff
@@ -0,0 +1,421 @@
+// Crest Ocean System
```

Comment:
I assume the XYZ.Effect.cs naming is standard? have not come across it before so just curious

Reply:
I don't think there is an official standard. But Unity does this in Graphics repository and apparently Microsoft does it too.

Reply:
ok gotcha.. if we're going with this i think we shoudl add this to our coding standard - in which case i'd like to understand when to use it and how. my gut tells me tha the first part of the name should perhaps match the namespace, but im just guessing.

but if we do introduce it to our standards there will be scripts that dont conform, and renaming scripts is ugly, so i think im going to be boring and vote to keep it consistent with how we usually do it - simply UnderwaterRendererEffect, unless that feels bad.

Reply:
We wouldn't have to change anything since it is only for partial classes which is also how Unity uses it. It isn't possible to include namespaces as Unity requires component names and file names to match exactly.

Reply:
I'm happy to go with whichever you want to though.

Reply:
oh so its for partial classes. that works great. approved!